src/lightkurve/interact.py

```diff
@@ -507,15 +509,78 @@ def add_gaia_figure_elements(tpf, fig, magnitude_limit=18):
 
     # mark the target's position too
     target_ra, target_dec, pm_corrected = _get_corrected_coordinate(tpf)
+    target_x, target_y = None, None
     if target_ra is not None and target_dec is not None:
-        target_x, target_y = tpf.wcs.all_world2pix([(target_ra, target_dec)], 0)[0]
-        fig.cross(x=tpf.column + target_x, y=tpf.row + target_y, size=20, color="black", line_width=1)
+        pix_x, pix_y = tpf.wcs.all_world2pix([(target_ra, target_dec)], 0)[0]
+        target_x, target_y = tpf.column + pix_x, tpf.row + pix_y
+        fig.cross(x=target_x, y=target_y, size=20, color="black", line_width=1)
         if not pm_corrected:
             warnings.warn(("Proper motion correction cannot be applied to the target, as none is available. "
                            "Thus the target (the cross) might be noticeably away from its actual position, "
                            "if it has large proper motion."),
                            category=LightkurveWarning)
 
+    # display an arrow on the selected target
+    arrow_head = VeeHead(size=16)
+    arrow_4_selected = Arrow(end=arrow_head, line_color="red", line_width=4,
+                             x_start=0, y_start=0, x_end=0, y_end=0, tags=["selected"],
+                             visible=False)
+    fig.add_layout(arrow_4_selected)
+
+    def show_arrow_at_target(attr, old, new):
+        if len(new) > 0:
+            x, y = source.data["x"][new[0]], source.data["y"][new[0]]
+
+            # workaround: the arrow_head color should have been specified once
+            # in its creation, but it seems to hit a bokeh bug, resulting in an error
+            # of the form  ValueError("expected ..., got {'value': 'red'}")
+            # in actual websocket call, it seems that the color value is
+            # sent as "{'value': 'red'}", but they are expdecting "red" instead.
+            # somehow the error is bypassed if I specify it later in here.
+            #
+            # The issue is present in bokeh 2.2.3 / 2.1.1, but  not in bokeh 2.3.1
+            # I cannot identify a specific issue /PR on github about it though.
```

Comment:
I'd be open to update the Bokeh dependency to be e.g. `bokeh>=2.3.1`, because Bokeh v1.1 is already three years old at this stage, or do we think supporting older Bokeh versions makes dependency issues much easier for our users?

Reply:
I don't think we can require `bokeh>=2.3.1` yet, until #990 is resolved (pixel selection in `tpf.interact()` does not work for bokeh>=2.3`